src/js/components/landing/BlockContent.jsx

```diff
@@ -22,21 +22,8 @@ export default class BlockContent extends React.Component {
         let thirdBlock = null;
 
         if (this.props.type === 'home') {
-            if (this.props.session.user.helpOnly) {
```

Comment:
If they have helpOnly permissions, they should never get to the "home" page in the first place.